doc/rst/tools/unstableWarningAnonymizer/unstableWarningAnonymizer.rst

```diff
@@ -0,0 +1,185 @@
+
+Next, run the built script over your output file. For our convenience, it
+would be helpful to run with the `--csv` and `--sorted` flags, or `-c`
+and `-d` if you want to use the shorter version of those flags.
+
+.. code-block:: sh
+
+  ./unstableAnonScript --csv --sorted --inputFiles myUnstableOutput.txt --outputFile mySummary.csv
+
+or
+
+.. code-block:: sh
+
+  ./unstableAnonScript -c -d -i myUnstableOutput.txt -o mySummary.csv
+
+Note that `--inputFiles`/`-i` can take multiple files, so if you have
+multiple chapel programs you'd like to share the results for, you
+can combine the results together by specifying the unstable warnings from
+all of those programs at the same time:
+
+.. code-block:: sh
+
+  ./unstableAnonScript -c  -d -n -i myUnstableOutput1.txt myUnstableOutput2.txt myUnstableOutput3.txt -o mySummary.csv
+
+After any of these commands, `mySummary.csv` (or whatever you've named
+it) should contain a comma-separated list of the unstable warnings
+generated by your program(s) and their counts, sorted from most common
+to least. There should be no identifying information in this file, so
+at this point, it should be safe to send the file to us.
+
+You could also additionally run with the `--numFiles` flag (`-n` for
+short), which will include the number of different files where each
+unstable warning was generated:
+
+
+.. code-block:: sh
+
+ ./unstableAnonScript --csv --sorted --numFiles --inputFiles myUnstableOutput.txt --outputFile mySummary.csv
+
+or
+
+.. code-block:: sh
+
+  ./unstableAnonScript -c -d -n -i myUnstableOutput.txt -o mySummary.csv
+
+This information would be helpful for our metrics, but is not essential.
+
+
+For a full list of options, run the script with the `--help` flag:
+
+.. code-block:: sh
+
+  ./unstableAnonScript --help
+  USAGE: unstableAnonScript [-h, --help] [-c, --csv] [-n, --numFiles] [-d, --sorted] [-i, --inputFiles <INPUTFILE> ...] [-o, --outputFile <OUTPUTFILE>] [-x, --topX <TOPX>]
+
+  OPTIONS:
+    -h, --help                    Display this message and exit
+    -c, --csv                     Write the output in csv format.
+                                  Defaults to false, which writes in a pretty format
+    -i, --inputFiles <INPUTFILE>  The files containing the warnings. Defaults to stdin
+    -n, --numFiles                Show the number of unique chapel files that had each warning.
+                                  Defaults to false
+    -o, --outputFile <OUTPUTFILE> The file to write the output to. Defaults to stdout
+    -d, --sorted                  Sort the output by descending frequency of each warning.
+                                  Defaults to false, which sorts by the warning message
+    -x, --topX <TOPX>             Show the top X most frequent warnings. Implies --sorted.
+                                  Defaults to showing all warnings.
+                                  In case of a tie for the last place, the warning that comes first in the sorted list is chosen
```

Comment:
I'd suggest not pasting the output of `--help` here, that way its less likely to get out of sync with the code. I'd just say ``for a full list of options, run `./unstableAnonScript --help` `` or something like that

Reply:
I think it is useful for a user to look at the full list of options to know what the script can do without us asking them to build and run it. But this help message also exists in the description comment in the script itself, so I will remove it from here, and arguably a comment within a file is less likely to fall out of sync.